nengo_gui/static/datastore.js

```diff
@@ -127,7 +121,6 @@ Nengo.DataStore.prototype.is_at_end = function() {
     return (ts.last_time < ts.first_shown_time + ts.shown_time + 0.000000001);
```

Comment:
Not part of this PR, but as it allows me to comment on code, I am misusing it for this. I think it is more readable to write `1e-9` than `0.000000001` (if I counted correctly). But I'm also wondering why has this magic constant to be added in the first place (and I don't comletely get the logic of this function).